### PR TITLE
Dw 458 slider with real data

### PR DESCRIPTION
**Pull request by lucianaDilizio**
PR based on @cbernat's one. (https://github.com/FromDoppler/doppler-webapp/pull/1086)
Based on what she was working, I have added changes to make it work. 

To test it: https://cdn.fromdoppler.com/doppler-webapp/demo-build2740/#/plan-selection

image

### CODE REVIEW

File: src/components/ChangePlan/PlanCalculator.js [hunk 1]
```diff
@@ -7,6 +7,25 @@ import queryString from 'query-string';
 import { extractParameter, getPlanFee } from '../../utils';
 import { useRouteMatch, Link } from 'react-router-dom';
 
+const NavigatorTabs = ({ tabs, pathType, selectedPlanType }) => {
+  return (
+    <nav className="tabs-wrapper">
+      <ul className="tabs-nav" data-tab-active="1">
```
Comment (cbernat) [inline src/components/ChangePlan/PlanCalculator.js:13]: Here I imagine that `data-tab-active` would shift accordingly with type plan selected by url, if any.

Comment (lucianaDilizio) [reply src/components/ChangePlan/PlanCalculator.js:13]: I didn't realize about this. I will check it 👍

Comment (lucianaDilizio) [reply src/components/ChangePlan/PlanCalculator.js:13]: I think that data-tab-active is not necessary.

Comment (lucianaDilizio) [reply src/components/ChangePlan/PlanCalculator.js:13]: I have to make changes to apply UI, so I can fix this there.

**Comment by cbernat** [comment]
I've tested and it's working very nice. Good job @lucianaDilizio ! 🚀

File: src/components/ChangePlan/PlanCalculator.js (hunk 1 above)
Comment (cbernat) [reply src/components/ChangePlan/PlanCalculator.js:13]: 👍

**Comment by cbernat** [comment]
@lucianaDilizio Only one issue, please check commit names. Perhaps prettier commits could be a fixup for previous commit. It would be very nice that we could read the functionality through commit titles.

**Pull request merged**